lnbits/templates/base.html

```diff
@@ -23,6 +23,57 @@
       .wallet-list-card:first-child {
         margin-left: 1px;
       }
+      /* Force chip content to wrap on mobile */
+      @media (max-width: 1023px) {
+        .q-chip {
+          height: auto !important;
+          min-height: 2rem !important;
+        }
+        .q-chip .q-chip__content {
+          white-space: normal !important;
+          word-break: break-all !important;
+          min-height: 1.5rem !important;
+          height: auto !important;
+        }
+        .q-chip .ellipsis {
+          overflow: visible !important;
+          text-overflow: clip !important;
+          white-space: normal !important;
+          word-break: break-all !important;
+        }
+        /* Fix q-input hint text wrapping and container growth */
+        .q-field {
+          overflow: visible !important;
+          height: auto !important;
+          min-height: 56px !important;
+        }
+        .q-field__inner {
```

Comment:
this introduced another bug on the register account page. see screenshot. 

![Image](https://github.com/user-attachments/assets/115261b2-9072-4c5a-886b-dc3b6b023e39)